lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -375,6 +372,31 @@ protected function _getCreateTableSQL($name, array $columns, array $options = []
         return $query;
     }
 
+    /**
+     * Generate a PRIMARY KEY definition if no autoincrement value is used
+     *
+     * @param string[] $columns
+     * @param mixed[] $options
+     *
+     * @return string
+     */
+    private function getNonAutoincrementPrimaryKeyDefinition(array $columns, array $options)
+    {
+        if (! isset($options['primary']) || empty($options['primary'])) {
```

Comment:
The combination of `!isset() || empty()` is redundant. It's covered by `empty()`.